### ISSUE TITLE
Add CycloneDX SBOMs example for version supporting 1.5 and 1.6

### ISSUE THREAD

**Issue by viveksahu26** [CycloneDX/bom-examples#44]
Currently there are are [SBOM](https://github.com/CycloneDX/bom-examples/tree/master/SBOM) CycloneDX examples of version <1.4. It would be great to add more examples supporting CycloneDX version 1.5 and 1.6.

**Comment by stevespringett** [comment]
Are there any specific features of v1.5 or 1.6 that you'd like to see in the SBOM examples?

**Comment by viveksahu26** [comment]
Yes, Basically In version [1.5 ](https://cyclonedx.org/docs/1.5/json/#metadata_tools)and [1.6](https://cyclonedx.org/docs/1.6/json/#tab-pane_metadata_tools_oneOf_i0), there are 2 new section `components` and `services` added under  `Metadata --> Tools`. Whereas in [version 1.4](https://cyclonedx.org/docs/1.4/json/#metadata_tools) it doesn't have those section inside `Metadata --> Tools `. So, i am bit curious to see those fields.

**Comment by jkowalleck** [comment]
> Yes, Basically In version [1.5 ](https://cyclonedx.org/docs/1.5/json/#metadata_tools)and [1.6](https://cyclonedx.org/docs/1.6/json/#tab-pane_metadata_tools_oneOf_i0), there are 2 new section `components` and `services` added under `Metadata --> Tools`. Whereas in [version 1.4](https://cyclonedx.org/docs/1.4/json/#metadata_tools) it doesn't have those section inside `Metadata --> Tools `. So, i am bit curious to see those fields.

see here for CDX 1.6: 
https://github.com/CycloneDX/cyclonedx-python/blob/main/tests/_data/snapshots/environment/pep639-texts_with-license-pep639_1.6.xml.bin#L4-L52
versus 1.4:
https://github.com/CycloneDX/cyclonedx-python/blob/main/tests/_data/snapshots/environment/pep639-texts_with-license-pep639_1.4.xml.bin#L4-L42
